### ISSUE TITLE
input in multi lines brokes

### ISSUE THREAD

**Issue by kurokirasama** [nushell/nushell#11341]
### Describe the bug

When using `input`, and the text goes to the next line, weird things happens, kind of the line cloning itself. I have a video for better understanding

### How to reproduce

```
input "enter enough text to go to the next line"
```

### Expected behavior

allow to write multiline text

### Screenshots

https://github.com/nushell/nushell/assets/5784951/032ba44a-c8ba-4465-a201-e898037c6380

### Configuration

| key                | value                                                                                                                                                                                               |
| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| version            | 0.88.1                                                                                                                                                                                              |
| branch             | main                                                                                                                                                                                                |
| commit_hash        | fd56768fdcc1acbebcb75745600fc7419f72833d                                                                                                                                                            |
| build_os           | linux-x86_64                                                                                                                                                                                        |
| build_target       | x86_64-unknown-linux-gnu                                                                                                                                                                            |
| rust_version       | rustc 1.72.1 (d5c2e9c34 2023-09-13)                                                                                                                                                                 |
| rust_channel       | 1.72.1-x86_64-unknown-linux-gnu                                                                                                                                                                     |
| cargo_version      | cargo 1.72.1 (103a7ff2e 2023-08-15)                                                                                                                                                                 |
| build_time         | 2023-12-14 15:38:19 -03:00                                                                                                                                                                          |
| build_rust_channel | release                                                                                                                                                                                             |
| allocator          | mimalloc                                                                                                                                                                                            |
| features           | dataframe, default, extra, sqlite, trash, which, zip                                                                                                                                                |
| installed_plugins  | custom-value generate, custom-value generate2, custom-value update, from eml, from ics, from ini, from vcf, gstat, hist, inc, net, plot, port scan, query, query json, query web, query xml, xyplot |

### Additional context

video example of an attempt of a cli chat, using input to obtain the user input

**Comment by fdncred** [comment]
it doesn't seem to be multi line but rather when the line wraps at the edge of the screen, right?

**Comment by kurokirasama** [comment]
I mean, from this, right before the end of the line:
![input1](https://github.com/nushell/nushell/assets/5784951/3962dd13-8832-4971-a766-8cac94803cfa)

go to this:
![input2](https://github.com/nushell/nushell/assets/5784951/20086409-f049-49ec-bf9f-0d2940663973)
one new line for every written char

**Comment by fdncred** [comment]
right, once it wraps at the screen edge, the entire line is duplicated for every character. i agree it's broken but it's not like you have text on multiple lines that you're dealing with, like:
```
line1
line2
line3
```

**Comment by kurokirasama** [comment]
right, but `alt+enter` doesn't work when writing in `input`, it just send the `Enter`. Is that another bug?

**Comment by fdncred** [comment]
input isn't meant to capture multi-line text, so if that's what you want, i don't think it's built into it. what i'm saying is that the text wrapping shouldn't create another line for each character typed.

**Comment by devurandom** [comment]
A feature request for multi-line text input would be https://github.com/nushell/nushell/issues/11376.

**Comment by iesahin** [comment]
Have this bug as well. `input` duplicates the previous lines when the text reaches to the terminal border.

I can spend some time with this bug. Where can I start?

**Comment by fdncred** [comment]
I'm not sure. It could be in nushell or reedline or something in the input command.

**Comment by flovilmart** [comment]
> switch to our own bugs from reedline

That's a neat way to put it!

**Comment by ItsHarper** [comment]
Version 106.1 (which includes #15369) still suffers from this bug unless you specify the `--reedline` option when calling `input`.

**Comment by sholderbach** [comment]
When we can make everyone happy with the `--reedline` behavior we may want to deprecate the flag and switch to that implementation. 

Not stopping anyone if they try to fix the current implementation. But from a long term perspective I would like to reduce the complexity.

**Comment by flovilmart** [comment]
@sholderbach  I’d be happy to promote the feature flag. We know some cases that are not suitable for `reedline` , they’re properly implemented now through the backwards compatibility. 
What would be the proper release procedure?
